collaboration/README.md

```diff
@@ -1,5 +1,42 @@
 # Collaboration
-
```

Comment:
Hi Muna!
1. Line 2: Replace the single dash with `---` if it is meant to be a horizontal rule, or add text to make it a valid list item.